app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/shared/Attribute.js

```diff
@@ -1,4 +1,4 @@
-import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
+import { faInfoCircle, faPencil } from '@fortawesome/free-solid-svg-icons'
```

Comment:
I think `faMagic` is a better fit.
```suggestion
import { faInfoCircle, faMagic } from '@fortawesome/free-solid-svg-icons'

Reply:
That suits the function even better, thanks for the suggestion. Updated.